ci/autoscaler/pipeline.yml

```diff
@@ -500,6 +500,10 @@ jobs:
     params:
       UPLOADER_KEY: ((autoscaler_blobstore_uploader_key))
       GITHUB_TOKEN: ((autoscaler_access_token))
+      AUTOSCALER_CI_BOT_NAME: app-autoscaler-ci-bot
+      AUTOSCALER_CI_BOT_EMAIL: ApplicationAutoscaler@sap.com
```

Comment:
We could store `AUTOSCALER_CI_BOT_NAME` and `AUTOSCALER_CI_BOT_EMAIL` also in CredHub, but I decided to put them right into the pipeline for simplicity reasons.

Any reasons for not doing it like this?

Reply:
None, that I can think of.